be/src/exec/vectorized/hash_joiner.cpp

```diff
@@ -133,8 +133,13 @@ void HashJoiner::_init_hash_table_param(HashTableParam* param) {
     }
     param->predicate_slots = predicate_slots;
 
-    for (auto i = 0; i < _probe_expr_ctxs.size(); i++) {
-        param->join_keys.emplace_back(JoinKeyDesc{_probe_expr_ctxs[i]->root()->type().type, _is_null_safes[i]});
+    for (auto i = 0; i < _build_expr_ctxs.size(); i++) {
```

Comment:
originally, here is _probe_expr_ctxts, so is it a typo?

Reply:
which are the same

Reply:
The data types of build and probe are the same, but now I need Expr in Build, so I use BuildExpr instead